strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/nuget/NugetPackageController.java

```diff
@@ -388,7 +391,147 @@ private void getPackageInternal(String storageId,
             response.setStatus(INTERNAL_SERVER_ERROR.value());
         }
     }
+    
+    @ApiOperation(value = "Used to download the headers for a package")
+    @ApiResponses(value = { @ApiResponse(code = HttpURLConnection.HTTP_OK, message = "The package was downloaded successfully."),
+                            @ApiResponse(code = HttpURLConnection.HTTP_INTERNAL_ERROR, message = "An error occurred.") })
+    @PreAuthorize("hasAuthority('ARTIFACTS_RESOLVE')")
+    @RequestMapping(path = "{storageId}/{repositoryId}/{commandName:(?:download|package)}/{packageId}/{packageVersion}", method = RequestMethod.HEAD, produces = MediaType.APPLICATION_OCTET_STREAM)
+    public void downloadPackageHeaders(@ApiParam(value = "The storageId", required = true) @PathVariable(name = "storageId") String storageId,
```

Comment:
This method should be called `getPackageHeaders`, as you're not downloading headers, you're getting them.